students/Maxim_Kliashchou/HomeWork03/Task04/by.maxim.kliashchou/java/service/EncoderService.java

```diff
@@ -0,0 +1,11 @@
+package service;
+
+public class EncoderService {
+    public void encode(short code){
```

Comment:
short is a bad choice to decode char to.
Could you tell me why?

Reply:
I don't know. Could you tell me why? @javaGuruBY